server/src/main/java/fr/uga/l3miage/tp4/models/CandidateEvaluationGridEntity.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+public class CandidateEvaluationGridEntity {
+    @Id
+    private Long sheetNumber;
+    @Column(precision = 2)
+    private double grade;
+    private LocalDateTime submissionDate;
+
+    @ManyToOne
+    private ExaminerEntity examinerEntity;
+
+    @ManyToOne
+    private CandidateEntity candidateEntity;
+
+    @ManyToOne
+    private ExamEntity examEntity;
+}
```

Comment:
Il manque une prise de recul, ici c'est plus logique que ce soit `CandidateEvaluationGridEntity ` qui porte ses critéria 
```suggestion
    private ExamEntity examEntity;
    
    @ManyToMany
    private Set<EvaluationCriteriaEntity> evaluationCriteriaEntities;
}
```